code/modules/assembly/igniter.dm

```diff
@@ -43,6 +43,13 @@
 	sparks.start()
 	return TRUE
 
+/obj/item/assembly/igniter/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/assembly/igniter))
+		to_chat(user, "You briefly consider attaching [W] to [src], and then conclude that's a very silly thing to do and move on.")
+		return
+	..()
```

Comment:
return here probably?

Reply:
its the end of the proc the return is implicit

Reply:
also it's going to return in ..()

Reply:
no like return ..()

Reply:
so u get the return value of parent :)

Reply:
Wouldn't that then continue on to attach the igniter since the parent proc handles attachment?

Reply:
No cause you returned for that above

Reply:
the return is not implicit